client/devdocs/design/blocks.jsx

```diff
@@ -134,6 +135,7 @@ export default React.createClass( {
 					<DailyPostButton />
 					<PostLikes />
 					<ReaderFeaturedVideo />
+					<NpsSurvey />
```

Comment:
See [above](https://github.com/Automattic/wp-calypso/pull/12331/files#r107243023) -- I'd do the `isEnabled` check here.